pkg/asset/templates/kubeconfig.yaml

```diff
@@ -6,10 +6,11 @@ clusters:
     server: {{ .Server }}
```

Comment:
Drop .yaml from the template file name to correspond to dropping the extension everywhere else?